rss/index.py

```diff
@@ -18,70 +22,189 @@ def perform_search(archives: str, date_time: datetime.datetime) -> EPrintSet:
         The IDs of the archives to search
     date_time : datetime
         The date/time that defines the end of the 24 hour search window.
+    days : int
+        The number of days before the specified time for which to return records.
 
     Returns
     -------
-    hits : List
-        The results of the Elasticsearch search as a list of .
+    status : int
```

Comment:
Is this the status code that we are returning? If so, we should leave this concern to the controller. The controller is responsible for deciding what to return to the client, including status code and response headers.

Reply:
Now that you mention this it is obvious!  I made this change where needed in index as well as in the serializers.

Reply:
Looks good